docs/linking/4-grant-consent.puml

```diff
@@ -0,0 +1,63 @@
+@startuml
+
+!pragma teoz true
+
+title PISP Linking: Grant consent
+
+participant "PISP" as PISP
+
+box "Mojaloop"
+    participant Switch
+end box
+
+participant "DFSP" as DFSP
+
+autonumber 1 "<b>GRANT-#</b>"
+
+DFSP -> Switch ++: ""POST /consents {""\n\
+    ""  id: "123",""\n\
+    ""  requestId: "456",""\n\
+    ""  initiatorId: "PISP",""\n\
+    ""  participantId: "DFSP",""\n\
+    ""  scopes: {""\n\
+    ""    accounts.getBalance: "12345-67890",""\n\
+    ""    accounts.transfer: "12345-67890",""\n\
+    ""  }""\n\
+    ""  credential: null""\n\
+    ""}""
+Switch --> DFSP: ""202 Accepted""
+deactivate DFSP
+
+note over Switch
+  The Switch must inform both the PISP and the Auth service about the consent.
```

Comment:
This is somewhat up for debate (see https://github.com/mojaloop/pisp/issues/36), as the switch has no method at this stage to "duplicate" these messages, so we might need the DFSP to send 2 separate `POST /consents`, one to the PISP and another to the auth-service.

Or we can give that responsibility to the auth-service